### PR TITLE
Enhancement/user query

### LINKED ISSUE #6: User Dashboard
When I as a user on the Dashboard when I click on upcoming event a module should open with the event details
- Should be able to click on a family and be redirected to their profile
- Should be able to press rsvp (if not already going) 
- Should be ablt to cancel rsvp (if already going)

### LINKED ISSUE #34: Change userProfile to populate the event component

### PR DESCRIPTION
**Pull request by tfoxcollis**
## Summary:
Update user query to work with backend API.
Add event query to work with backend API.
Update views so that event card "view details" button is clickable and modal pops up.
Update map markers to make the modal pop up.
User profile refactor so it uses the Events component.

## Type of Changes:
- [ ] Bug fix
- [x] Refactor
- [x] New feature

## How was this tested:
Chrome Dev Tools

## What are the relevant tickets:
closes #34
closes #6

### CODE REVIEW

**Review by abodnar1 — COMMENTED** [review]
Looks good!

File: src/Components/UserProfile/UserProfile.js [hunk 1]
```diff
@@ -3,44 +3,31 @@ import { useQuery } from "@apollo/client";
...
 import EventModal from '../EventModal/EventModal'
-import EventCard from "../EventCard/EventCard";
+import Events from "../Events/Events"
 
 // SETUP AS A FAMILY VIEW FROM THE EVENT DETAILS PAGE
 // Can use a query hook for data for DRY code or just pass as props
 
 const UserProfile = () => {
+  const { loading, error, data } = useQuery(GET_USER_BY_ID, {
+    variables: {"id": process.env.REACT_APP_USER_ID}
```
Comment (BlueNealis) [inline src/Components/UserProfile/UserProfile.js:13]: I cant believe how simple this looks once the query is all set up this is awesome!

File: src/queries.js [hunk 2]
```diff
@@ -1,73 +1,59 @@
...
-  zipCode
-  rsvpEvents {
+  user(id: $id) {
     id
-    title
+    userName
+    email
     description
+    image
     zipCode
```
Comment (BlueNealis) [inline src/queries.js:12]: Nice and here are the updated queries!

**Review by BlueNealis — APPROVED** [review]
Wow you really did so many things thats awesome Thank you!

**Pull request merged**